### ISSUE TITLE
Open, enter login and it crashed

### ISSUE THREAD

**Issue by chakzefir** [avocode/avocode#376]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.0-91b4012
Source UID: undefined
```
TypeError: undefined is not a function
  at Object.module.exports._parseCookie (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/auth/lib/utils/api.js:1:655)
  at Request._callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/auth/lib/utils/api.js:1:1155)
  at Request.self.callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/jsdom/node_modules/request/request.js:198:22)
  at emitTwo (events.js:87:13)
  at Request.emit (events.js:169:7)
  at Request.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/jsdom/node_modules/request/request.js:1057:14)
  at emitOne (events.js:82:20)
  at Request.emit (events.js:166:7)
  at IncomingMessage.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/jsdom/node_modules/request/request.js:1003:12)
  at emitNone (events.js:72:20)
  at IncomingMessage.emit (events.js:163:7)
  at _stream_readable.js:891:16
  at process._tickCallback (node.js:357:13)

```

**Comment by rychlis** [comment]
thanks for the report, what does "crashed" mean? Did Avocode close after this error? Did you use the older version before this one or are you a new user?

**Comment by rychlis** [comment]
OK I can reproduce the issue now, it only happens on brand new installation. You can workaround it by simply restarting the App, we will release a fixed version soon

**Comment by rychlis** [comment]
We've released Avocode 2.4.1 that fixes this issue. Thanks for the report

**Comment by chakzefir** [comment]
thank you, sorry for not answering so long, but now there is no such bugs. thanks again
